include/slb/detail/config.hpp

```diff
@@ -27,4 +27,24 @@
 #define __has_feature(x) 0
 #endif
 
+// libstdc++ did not mark `operator()` as `noexcept` until version 8.
```

Comment:
Same thing for `operator value_type`. We should provide that one too (with tests), but it could be handled by a separate PR.